Source/ProgressionSystemRuntime/Public/Data/PSTypes.h

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#pragma once
+
+#include "Bomber.h"
+#include "Structures/PlayerTag.h"
+#include "PSTypes.generated.h"
+
+/**
+ * Basic structure for all save data information regarding the progression.
+ * Same structure will reflected in the save file. Initial load performed based on the data in the DT Table 
+ */
+USTRUCT(BlueprintType)
+struct FPSRowData : public FTableRowBase
```

Comment:
`FTableRowBase` is never included there, so it will fail the cook. Please include `#include "Engine/DataTable.h"`

Reply:
added.

MacOS support is quite hard for me. I do have mac but  hardly limited with disk space and always have to download new verison of UI. I'll try to